CONTRIBUTING.md

```diff
@@ -1,75 +1,76 @@
-Contributing to KEDA Helm Charts
-===
+# Contributing to KEDA Helm Charts
 
 Thanks for helping making KEDA better!
 
 ## Shipping a new version
 
 You can easily release a new Helm chart version:
 
-1. Update the version of the Helm chart in `Chart.yaml`
-2. Package the Helm chart 
-   - For KEDA:
+1. Update the version of the Helm chart in `Chart.yaml`.
+1. Package the Helm chart
+    - For KEDA:
+        ```shell
+        $ helm package keda --destination docs
+        Successfully packaged chart and saved it to: docs/keda-2.4.0.tgz
+        ```
+    - For HTTP Addon:
+        ```shell
+        $ helm package http-add-on --destination docs
+        Successfully packaged chart and saved it to: docs/keda-add-ons-http-0.2.0.tgz
+        ```
+    - For KEDA external scaler for Azure Cosmos DB:
+        ```shell
+        $ helm package cosmosdb-scaler --destination docs
+        Successfully packaged chart and saved it to: docs/keda-external-scaler-azure-cosmos-db-0.1.0.tgz
```

Comment:
This does not seem correct as the output name should use the name of the folder.

That said, we should rename the folder as well:
```suggestion
        $ helm package azure-cosmos-db-add-on --destination docs
        Successfully packaged chart and saved it to: docs/azure-cosmos-db-add-on-0.1.0.tgz
```